exasol_transformers_extension/udfs/models/translation_udf.py

```diff
@@ -0,0 +1,188 @@
+import torch
+import pandas as pd
+import transformers
+from typing import List
+from exasol_transformers_extension.deployment import constants
+from exasol_transformers_extension.utils import device_management, \
+    dataframe_operations, bucketfs_operations
+
+
+class TranslationUDF:
+    def __init__(self,
+                 exa,
+                 batch_size=100,
+                 pipeline=transformers.pipeline,
+                 base_model=transformers.AutoModelForSeq2SeqLM,
+                 tokenizer=transformers.AutoTokenizer):
+        self.exa = exa
+        self.batch_size = batch_size
+        self.pipeline = pipeline
+        self.base_model = base_model
+        self.tokenizer = tokenizer
+        self.device = None
+        self.cache_dir = None
+        self.last_loaded_model_key = None
+        self.last_loaded_model = None
+        self.last_loaded_tokenizer = None
+        self.last_created_pipeline = None
+        self.translation_prefix = "translate {src_lang} to {tgt_lang}: "
```

Comment:
Yeah. According to multi-lang models, it may be necessary to call API  differently. For example:

While it is necessary to give src_lang/tgt_lang parameters in the __call__ method of the m2m100 model, these parameters do not work in the t5 model. 

I've organized it according to the most common usage

Reply:
ok, that is fine for now, but maybe we can determine the common ways and add a parameter to switch the behavior, but I would move this to a ticket. Further, we need to document this assumption to the user, by stating which models are compatible and which method for encoding the language we are using